frontend/public/locales/en/apply.json

```diff
@@ -424,7 +415,7 @@
       },
       "how-we-protect-your-privacy": {
         "heading": "How we protect your privacy",
-        "personal-information-collection-and-use-authorization": "The collection and use of your personal information is authorized under the <i>Privacy Act</i>, the <i>Department of Health Act</i>, the <i>Dental Care Measures Act</i>, the <i>Department of Employment and Social Development Act</i>, and other applicable laws.",
+        "personal-information-collection-and-use-authorization": "The collection and use of your personal information is authorized under the <i>Privacy Act</i>, the <i>Department of Health Act</i>, the <i>Dental Care Measures Act</i>, the <i>Department of Employment and Social Development Act</i>, and other applicable laws. ",
```

Comment:
Was the added space here intentional?

Reply:
And I think the `<i>` elements should be `<em>`, shouldn't they?

Reply:
Removed spaces, and extended to italics to french translations as well

Reply:
I guess we can keep the `<i>` for now.. According to the docs, the default set of allowed tags are `['br', 'strong', 'i', 'p']`